scripts/pre_commit_hook.py

```diff
@@ -191,7 +192,7 @@ def check_changes_in_config() -> None:
 
 def run_prettier() -> None:
     """Runs prettier formatter."""
-    subprocess.run([NPX_PATH, 'lint-staged'], check=True)
+    subprocess.run([NPX_CMD, 'lint-staged'], shell=False, check=True)
```

Comment:
You don't need the shell=False here, it is false by default. (I already fixed this in develop.)

Please remove -- thanks.

Reply:
Done.